framework/source/class/qx/html/Element.js

```diff
@@ -787,6 +795,17 @@ qx.Class.define("qx.html.Element",
       SUPPORT FOR ATTRIBUTE/STYLE/EVENT FLUSH
     ---------------------------------------------------------------------------
     */
+    
+    updateObjectId: function() {
+      // Copy Object Id
+      if (qx.core.Environment.get("module.objectid")) {
+        var id = null;
+        if (this.__widget && this.__widget.getObjectId()) {
+          id = qx.core.Id.getAbsoluteIdOf(this.__widget, false) || null;
+        }
+        this.setAttribute("data-object-id", id, true);
```

Comment:
maybe namespace this with "data-qx-object-id" ?

Reply:
done